app/src/main/java/com/example/parkingsystem/mvp/view/ParkingView.java

```diff
@@ -0,0 +1,38 @@
+package com.example.parkingsystem.mvp.view;
+
+import android.app.Activity;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.example.parkingsystem.R;
+import com.example.parkingsystem.databinding.ActivityMainBinding;
+
+public class ParkingView extends ActivityView {
+
+    private final ActivityMainBinding binding;
+
+    public ParkingView(Activity activity, ActivityMainBinding binding){
+        super(activity);
+        this.binding = binding;
+    }
+
+    public String getSizeSubmitted(){
+        String size="";
+            size = binding.quantityParkingLots.getText().toString();
```

Comment:
Variable definition and assignment can be done in a single line.

Reply:
updated method getSizeSubmitted()
![image](https://user-images.githubusercontent.com/85253636/122767839-6efecc00-d279-11eb-8c7c-277f535c9863.png)